Bricks/template/decay.h

```diff
@@ -72,8 +72,8 @@ struct rmref_impl<std::tuple<TS...>&&> {
   typedef metaprogramming::map<impl, std::tuple<TS...>> type;
 };
 
-template <typename... TS>
-using rmref = typename rmref_impl<TS...>::type;
+template <typename T>
+using rmref = typename rmref_impl<T>::type;
```

Comment:
I believe so -- it only uses `<TS...>` as parts of tuples in specializations. And top-level `make test` and `make check` passed on Linux.